gameofgame.py

```diff
@@ -20,39 +20,50 @@
 def choose_country():
     country = random.choice(list(countries_capitals.keys()))
     return country
+    
 
 # Function to check if the guess is correct
 def check_guess(guess, country):
     capital = countries_capitals[country]
     if guess.lower() == capital.lower():
         return True
     else:
-        return False
+       return False
 
 # Main game loop
 while True:
     # Choose a random country
     country = choose_country()
 
-    # Player 1's turn
+    # Both Players take turns to play
+    #player1 goes first
     print("Player 1's turn")
-    guess = input(f"What is the capital of {country}? ")
-    if check_guess(guess, country):
-        print("Correct!")
-    else:
-        print("Incorrect.")
+    player1_guess = input(f"What is the capital of {country}? ")
+    
+    ##player2
+    print ("player 2 's turn")
```

Comment:
Here the formatting of the message text is different from player 1. There is no capitalisation, and there is a space before the apostrophe.